README.adoc

```diff
@@ -1833,6 +1833,60 @@ storages:
     url: https://exstorage2/api/webdav
 ----
 
+=== Multi-Fairspace metadata views integration
+
+Fairspace can have only single data model configured. However, if there is a need to have data from multiple models,
+representing multiple domains that would be combined combined in single user interface, it is possible to integrate multiple Fairspace instances together.
+
+
+Fairspace (main) instance can be configured as a view interface of metadata from another (external) Fairspace instance.
+External Fairspace can have a different metadata model configured than the main Fairspace.
+Important precondition is that the external instance has to be connected to the same Keycloak realm as the main instance.
+
+
+External metadata can be searched and browsed in the same way as the internal metadata views. When configured, there is an extra page,
+separate from internal metadata views, that allows to explore external metadata.
+Currently, cross-instance metadata search is not supported.
+
+
+For the external metadata pages, the tables and columns are created basing on the views configuration specified in that instance configuration.
+
+
+==== Access policy
+
+Connection to the same Keycloak realm allows to authenticate a user in all integrated Fairspace instances with a single set of login credentials.
+Each Fairspace instance is responsible for controlling the access to its own metadata and perform
+the required checks to ensure that users only get to see metadata of an instance, if has a view public metadata role assigned within that instance.
+
+==== Configuration
+
+Multiple external Fairspace metadata pages can be configured simultaneously. A list of configuration parameters is presented below.
+
+|===
+| Parameter | Description
+
+| ``name``
+| Unique name of the metadata source.
+| ``label``
+| String to be used as a display name of the metadata source.
+| ``url``
+| Fairspace instance to connect to.
+|===
+
+Sample configuration of external metadata sources in YAML format:
+[source, bash]
+----
+metadata-sources:
```

Comment:
So this is in values.yaml? And do you also need to add the local instance, or only remotes?

Reply:
only remotes 
I forgot to add it to Kubernetes/Helm config, will do!